packages/core/src/lib/connector-info.ts

```diff
@@ -0,0 +1,16 @@
+import { getConnectorInstanceById } from '@/connectors';
+import { findAllConnectors, findConnectorById } from '@/queries/connector';
+
```

Comment:
Not sure whether should add a type `ConnectorInfo` here which is the union of all keys in both `Connector` and `ConnectorMetadata`.